mockingServer/modules/LoggingSession.py

```diff
@@ -0,0 +1,17 @@
+from mockingServer.modules.RequestHandler import RequestHandler
+from flask import Blueprint, make_response
+import json
+
+loggingBluePrint = Blueprint('LoggingService', import_name=__name__)
+
+
+class LoggingService(RequestHandler):
+    def post(self):
+        if not self.check_session_id():
+            return make_response('sessionID check failed', 401)
```

Comment:
Is this the real response?

Reply:
Yes, if you sent a request the message will be stored as a content attribute.
f.e: 
response = requests.post(...)
print(response.content) -> would provide this message.

Reply:
Funny, ok